GitUI/Translation/English.xlf

```diff
@@ -9203,6 +9203,10 @@ See the changes in the commit form.</source>
         <source>Show &amp;date column</source>
         <target />
       </trans-unit>
+      <trans-unit id="showGitNotesColumnToolStripMenuItem.Text">
+        <source>Show git &amp;notes column</source>
```

Comment:
The mnemonic conflicts with "Show git notes". This may be acceptable. I guess this will not be switched often anyway.